Caboodle/Browser/Browser.netstandard.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public partial class Browser
+    {
+#pragma warning disable CS1998
```

Comment:
You should just be able to omit `async` on these method declarations and then remove `#pragma`.